README.md

```diff
@@ -498,6 +506,54 @@ last_modified: Date;
 - GEOMETRY
 - JSON
 
+## Extended operators
+MySQL connector supports the following MySQL-specific operators:
+- [`match`](#operator-match)
+Please note extended operators are disabled by default, you must enable
+them at datasource level or model level by setting `allowExtendedOperators` to
+`true`.
+### Operator `match`
+The `match` operator allows you to perform a full text search using the [MATCH() .. AGAINST()](https://dev.mysql.com/doc/refman/8.0/en/fulltext-search.html) operator in MySQL.
+
+Three different modes of the `MATCH` clause are also available in the form of operators - 
+
+- `boolmatch` for [Boolean Full Text Search](https://dev.mysql.com/doc/refman/8.0/en/fulltext-boolean.html)
+- `nlmatch` for [Natural Language Full Text Search](https://dev.mysql.com/doc/refman/8.0/en/fulltext-natural-language.html)
+- `qematch` for [Full-Text Searches with Query Expansion](https://dev.mysql.com/doc/refman/8.0/en/fulltext-query-expansion.html)
+
+By default, the `match` operator works in Natural Language mode.
+
+**Note** The fields you are querying must be setup with a `FULLTEXT` index to perform full text search on them.
+Assuming a model such as this:
+```ts
+@model({
+  settings: {
+    allowExtendedOperators: true,
+  }
+})
+class Post {
+  @property({
+    type: 'string',
+    mysql: {
+      index: {
+        kind: 'FULLTEXT'
+      }
+    },
+  })
+  content: string;
+}
+```
+You can query the tags fields as follows:
```

Comment:
what do you mean by `tags fields` ?.

Reply:
Against mistake on my end, this should be `content`, not `tags`.